src/main/java/at/fhhagenberg/esd/sqe/ws20/gui/ECC.java

```diff
@@ -51,6 +51,11 @@ public void start(Stage stage) throws Exception {
 
         stage.setTitle(Messages.getString("ECC.2"));
         stage.setScene(scene);
+        
+        stage.heightProperty().addListener((obs, oldVal, newVal) -> {
```

Comment:
I refactored the controller in a way, that it can set the listener itself but it still needs the stage to do so. But should be a cleaner and more adaptable approach now.